attunement.lic

```diff
@@ -24,9 +25,13 @@ class Attunement
       return
     end
 
-    room_list = if @stationary_skills_only || DRStats.necromancer?
+    room_list = if @stationary_skills_only || (DRStats.necromancer? and !@force_power_walk)
```

Comment:
Id prefer the waggle set as well, making it optional cast. Some attunement rooms arent in justice (e.g. Shards rooms are all outside of town) so the eotb wouldn't be necessary.